### DISCUSSION TITLE
what mean "The dead letter producerName uses this format"??

### DISCUSSION THREAD

**Discussion by startjava** [apache/pulsar#22326] [General]
https://pulsar.apache.org/docs/3.2.x/concepts-messaging/#dead-letter-topic

The dead letter producerName uses this format:

<topicname>-<subscriptionname>-<consumername>-DLQ

**Comment by jak78** [comment]
Hey,
Pulsar messages have several metadata fields, including the name of the message producer. This metadata field is named `producerName`
The messages sent to the Dead Letter Queue also have a producer name, just like any other message.
The producer of these messages is named using the following format:
`<topicname>-<subscriptionname>-<consumername>-DLQ`
I hope this is clear now. Please let me know if you have any questions.
By the way, it might be necessary to clarify the docs here. @startjava are you willing to suggest an improvement? That would be amazing, and super useful to the community!
